packages/plugin-iteratively/package.json

```diff
@@ -34,11 +34,12 @@
     "url": "https://github.com/iterativelyhq/itly-sdk.git"
   },
   "dependencies": {
-    "@lukeed/uuid": "^2.0.0"
+    "uuid": "^8.3.2"
```

Comment:
@falconandy thanks for updating this. It has been giving some users trouble. Unfortunately the new lib is much bigger (3.3kb vs 333b) and we are hoping to keep the SDK as light as possible for the browser. The [uuid README.md](https://www.npmjs.com/package/uuid) says it should play nice with tree shaking. Could you please check the size difference in the webpack output between the two? I we may not need this to be an uuid, so depending on the size difference we might want to find another solution. Thank you.

https://bundlephobia.com/package/@lukeed/uuid@2.0.0
https://bundlephobia.com/package/uuid@8.3.2

Reply:
@justin-fiedler 
I did tests on simple application that uses SDK, Schema Validator and Iteratively plugins. Webpack mode is `production`

Results:
| Branch | Webpack Version | Output Size (KB) |
|-|-|-|
| ITLY-2380-webpack-5-browser-detection | 5.41.0 | 52.5 |
| ITLY-2380-webpack-5-browser-detection | 4.46.0 | 54 |
| master | 5.41.0 + node-polyfill-webpack-plugin | 503 |
| master | 4.46.0 | 56 |

Detailed report for the `ITLY-2380-webpack-5-browser-detection` branch and Webpack 5.41.0 - [report.zip](https://github.com/amplitude/itly-sdk/files/6732779/report.zip)

Reply:
This is great. Thank you.